backend/models/Message.js

```diff
@@ -3,7 +3,7 @@ const { Schema } = mongoose;
 
 const MessageSchema = new Schema({
   author: { type: Schema.Types.ObjectId, ref: "User", required: true },
-  chat: { type: Schema.Types.ObjectId, ref: "Chat", required: true },
+  chat: { type: Schema.Types.ObjectId, ref: "Chat", required: false },
```

Comment:
I think he left it because we would need to change all our previous test cases if we changed it, we probably need to make a separate issue for this.

Reply:
> I think he left it because we would need to change all our previous test cases if we changed it, we probably need to make a separate issue for this.

Makes sense